### ISSUE TITLE
Long table header repeated on each page.

### ISSUE THREAD

**Issue by rjazwiec** [rstudio/gt#1630]
## Prework

- [x] Read and abide by **gt**'s [code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html) and [contributing guidelines](https://github.com/rstudio/gt/blob/master/.github/CONTRIBUTING.md).
- [~x] Search for duplicates among the [existing issues](https://github.com/rstudio/gt/issues) (both open and closed).

## Proposal

As far as I looked it in .tex files gt uses longtable environment for tables spanning over few pages.
Unfortunately, table headers are not repeated on each page.

I'm not LaTex pro, but I tested manually adding `\endhead` after `\midrule` before the first row of data, and each time it worked very well. Both for tables rendered in portrait and in landscape*.

I think it would be nice to have parameter in gt() that decides if `\endhead` should be added to latex output.

*package `lscape`

**Comment by rjazwiec** [comment]
Sample result of manually adding `\endhead` rendered to pdf
[sample2.pdf](https://github.com/rstudio/gt/files/15073726/sample2.pdf)

**Comment by bzkrouse** [comment]
I agree this would be a very helpful addition!

**Comment by ujtwr** [comment]
I agree as well. Additionally, wouldn't it be useful if a label could be set at the top of the headers from the second page onward to indicate continuity, and that label could also be customizable?

The attached screenshot is from the 'kableExtra' package. In this package, there is a 'repeat_header' option that, when enabled, adds the label '(continued)' above the header on the second and subsequent pages. However, this label cannot be changed. As a Japanese user, there are times when I would like to change the label to Japanese :-) .


![image](https://github.com/user-attachments/assets/0a4761c6-a160-4b85-9e95-9a7244fad4a1)

**Comment by iantaylor-NOAA** [comment]
The `latex.header_repeat = TRUE` available since gt 1.1.0 (thanks to #1995 from @thebioengineer) solves this issue for me, so couldn't you close this issue (as well as #1982)? I got here by searching "how to make header repeat in gt long table" and was initially under the impression that the option wasn't yet available.

I agree that the suggestion for a customizable addition to the header like "(continued)" as suggested by @ujtwr is a good one, but that seems like it belongs in a separate issue.
